cmd/ipfs/init.go

```diff
@@ -90,6 +94,12 @@ environment variable:
 			return
 		}
 
+		keyType, _, err := req.Option("k").Int()
```

Comment:
I've used the long name for all the options that were using the short name in this file. 👍